herddb-utils/src/main/java/herddb/index/blink/BLink.java

```diff
@@ -1771,7 +1780,7 @@ public String toString() {
         /* *** FOR BOTH LEAVES AND NODES *** */
         /* ********************************* */
         boolean empty() {
-            return empty;
+            return outlink != null;
```

Comment:
I can't understand this.
Is this method used?

Reply:
Empty method is used during node search. Empty boolean flag wasn't needed because a node is empty if it was half-merged (and his outlink was set. A node with an outlink is a empty and reclamable node)